src/Razor/src/Microsoft.VisualStudio.RazorExtension/EmbeddedGrammars/aspnetcorerazor.tmLanguage.tmTheme

```diff
@@ -168,73 +169,76 @@
 
       <dict>
         <key>scope</key>
-        <string>constant.language, variable.language, storage.modifier.cs, keyword.type.cs, keyword.other.using.cs, keyword.other.await.cs, keyword.other.var.cs</string>
+        <string>keyword.control.conditional, keyword.control.loop, keyword.control.flow</string>
         <key>settings</key>
         <dict>
           <key>vsclassificationtype</key>
-          <string>Keyword</string>
+          <string>keyword - control</string>
         </dict>
       </dict>
 
       <dict>
         <key>scope</key>
-        <string>storage.type.cs, entity.name.type.class.cs</string>
+        <string>entity.name.label</string>
         <key>settings</key>
         <dict>
           <key>vsclassificationtype</key>
-          <string>class name</string>
+          <string>parameter name</string>
         </dict>
       </dict>
 
       <dict>
         <key>scope</key>
-        <string>entity.name.function.cs</string>
+        <string>entity.name.variable.parameter</string>
         <key>settings</key>
         <dict>
           <key>vsclassificationtype</key>
-          <string>method name</string>
+          <string>parameter name</string>
         </dict>
       </dict>
 
+      <!-- End  Generic colorization -->
+
+      <!-- Begin C# specific colorization -->
       <dict>
         <key>scope</key>
-        <string>variable.other.object.cs, entity.name.variable.local.cs</string>
+        <string>constant.language, variable.language, storage.modifier.cs, keyword.type.cs, keyword.other.using.cs, keyword.other.await.cs, keyword.other.var.cs</string>
         <key>settings</key>
         <dict>
           <key>vsclassificationtype</key>
-          <string>local name</string>
+          <string>Keyword</string>
         </dict>
       </dict>
 
-      <dict>
+      <!-- <dict>
```

Comment:
I know we didn't do it when we lobotomized TextMate before but would there be value in mentioning why we're commenting this out via a bug # or something?